### ISSUE TITLE
Does not work under MacOS 11.1

### ISSUE THREAD

**Issue by pagaille** [rusq/osx-callhistory-decryptor#2]
```
./callhistory 
MacOS X Call History Decryptor  © 2018 rusq
Based on Call History Decryptor © 2016 n0fate
Enter your account password to access keychain when prompted...
2021/02/26 11:33:40 *** filename: CallHistory.storedata
2021/02/26 11:33:40 no such table: ZCALLRECORD
```

**Comment by pagaille** [comment]
Discard this, I thought the app would automatically pick up the right file at the right location. One copied into the same folder, it works partially : 

- App begins to spit out the first calls but stops on this error : `sql: Scan error on column index 4, name "ZISO_COUNTRY_CODE": converting NULL to string is unsupported`
- History is not complete, but it looks like a MacOS issue since the same calls are reported in the FaceTime app.

**Comment by rusq** [comment]
@pagaille thank you, i'll have a look this weekend.

**Comment by rusq** [comment]
@pagaille decided not to wait till weekend, could you please check if it works for you now?

**Comment by pagaille** [comment]
Thanks ! Working ! 

Strangely I didn't succeed running the tool using the database file on its original location. I had to copy it first elsewhere.

Suggestions 👍 

1. Remove the trailing Z in the date time field (example I get : 2020-12-13 16:51:55Z), because Numbers can't cope with it. 
2. I wish the name of the callee or caller would have been displayed, based on my address book, but it might go further that what you intended.

Good job, thanks.

**Comment by rusq** [comment]
Thanks for checking and for the suggestions, - re (1) - i'll have a look.
As regards to the item 2, is out of scope for this project, but it's an interesting suggestion, maybe one sunny day I'll do it.

The reason you're unable to open the original file is that the database file is locked by the system:
```
2021/03/25 21:43:26 *** filename: /Users/rusq/Library/Application Support/CallHistoryDB/CallHistory.storedata
2021/03/25 21:43:26 database is locked
```
that's why, in [README.md][https://github.com/rusq/osx-callhistory-decryptor/blob/master/README.md#macos], it is suggested to make a copy.

**Comment by rusq** [comment]
Hey @pagaille, implemented your suggestion No. 1 : see pull #6 

TLDR; Use `-time-format` command line parameter to specify the format required.
